src/helpers/utils.ts

```diff
@@ -55,8 +55,8 @@ export const retryPost = async <T>(fn: () => Promise<any>, n: number): Promise<T
     } catch (err: any) {
       lastError = {
         isRetryError: true,
-        status: "SDK_RETRY_POST_ERROR",
-        message: err?.message ? err.message : JSON.stringify(err),
+        status: err.status,
```

Comment:
Updating the status with a proper status 500 from HTTP is a breaking change. If the function is successful, it will return the enclave status which is a string: STORESUCCESS, DATABASEFAILURE, KEYNOTEXIST (...) while it will be a Number in case of failure. The developer will have to filter the type if handling the response is needed.

Reply:
I have changed it so internal status is now a string on fail